config/pom.xml

```diff
@@ -115,6 +115,21 @@
                         </manifestEntries>
                     </archive>
                 </configuration>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>test-jar</goal>
+                        </goals>
+                        <configuration>
+                            <includes>
```

Comment:
```suggestion
                            <includes>
                                 <!-- Test classes exported for Jakarta extension reuse -->
```

Including an explanatory comment as to why we are creating these test-jars would prevent confusion about it in the future. I would suggest a start and end tag around the classes that are exported for the Jakarta extension specifically

Reply:
Liking this suggestion. And I think it's necessary too. For now this is a hand reach to make things work as duplicating the tests isn't nice either. But ideally we wouldn't have to do this for the actual test classes at all.

Furthermore, the utilities being shared should've been common already, although I feel those might better be placed in a separate module. As that's out of scope for this issue or Axon 4, this is fine for now. I'd hope we can adjust this in future major releases, though.